### PR TITLE
Use LSIO Apline Base as Dockerfile

### PR DESCRIPTION
**Pull request by ZachMyers3**
Hello,

Thought I'd attempt to contribute. Here's a dockerfile using lsio alpine base to run the nuxt.js generated static files.

Let me know what you think

### CODE REVIEW

**Comment by nahsi** [comment]
Why use nginx instead of caddy?

**Comment by ZachMyers3** [comment]
Mostly just familiarity. In my use-case this is passed publicly through traefik so I have no real preference what the reverse-proxy is within the container, just that the static files are served one way or another.

File: docker/nginx.conf [hunk 1]
```diff
@@ -0,0 +1,48 @@
+worker_processes 1;
+
+user abc abc;
+error_log /var/log/nginx/error.log warn;
```
Comment (nahsi) [inline docker/nginx.conf:4]: I think in official nginx image `/var/log/nginx/error.log` is a symlink to stdout.

It this the case for this image?

Comment (ZachMyers3) [reply docker/nginx.conf:4]: No

```
root@9257d5bc86eb:/# ls -lLi /var/log/nginx/error.log
6689757 -rw-r--r-- 1 root root 0 Sep 24 15:09 /var/log/nginx/error.log
```

**Pull request closed without merging**